index.d.ts

```diff
@@ -0,0 +1,5 @@
+export * from './advanced';
+
+import { QueryMiddleware } from './advanced';
+
+export const queryMiddleware: QueryMiddleware;
```

Comment:
This file also needs to export `networkInterfaces`, should it not?

Reply:
That should be covered by the `export *` from `advanced`

Reply:
There's a `const networkInterfaces` export also exported from `index.t.js` that's missing here.